roles/kube_nginx_ingress/tasks/main.yml

```diff
@@ -1,34 +1,22 @@
-- name: 'create tempfile for nginx-ingress values'
-  tempfile:
-    state: file
-    suffix: yml
-  register: nginx_ingress_value_file
-
-- debug:
-    var: nginx_ingress_value_file.path
-  when: debug|bool
-
-- name: 'copy nginx-ingress values into temporary file'
-  copy:
-    src: nginx_ingress_values.yml
-    dest: '{{ nginx_ingress_value_file.path }}'
 
 - name: 'install nginx ingress'
-  command: >-
-    {{ bin_dir }}/helm upgrade nginx-ingress
-    --install {{ nginx_ingress_release_name }}
-    --repo {{ nginx_ingress_repo }}
-    --version {{ nginx_ingress_version }}
-    --namespace {{ nginx_ingress_namespace }}
-    -f {{ nginx_ingress_value_file.path }}
+  helm_cli:
+    release: '{{ nginx_ingress_release_name }}'
+    chart:
+      name: nginx-ingress
+      version: '{{ nginx_ingress_version }}'
+      repo: '{{ nginx_ingress_repo }}'
+    namespace: '{{ nginx_ingress_namespace }}'
+    binary: '/usr/local/bin/'
+    values_list:
+    - "{{ lookup('file', role_path ~ '/files/nginx_ingress_values.yml') }}"
```

Comment:
Is there a way to include the YAML content of a file here, instead of the file as a string? This would cause Ansible to pre-parse the YAML, ruling out formatting issues at playbook init time rather than runtime.